### ISSUE TITLE
Quick installation fails

### ISSUE THREAD

**Issue by sgmiriuka** [egaffo/circompara2#1]
hi @egaffo 
installation file (install_circompara) is missing. Found it in bash folder, though fail when run. 
thnks
santiago

**Comment by egaffo** [comment]
Hi @sgmiriuka 
I fixed the path in the README.
What is failing in your installation? Do you have any error message?

Before running the install script, you need to install the following packages:
- git
- python2.7
- wget
- unzip
- pkg-config
- default-jre
- r-base-core
- libcurl4-openssl-dev
- libxml2-dev
- libssl-dev
- curl
- pigz
- python-is-python2
- python-dev-is-python2
(tested in a fresh Ubuntu 20.04).
I realize just now that I have missed declaring these dependencies in the installation guide.

Enrico

**Comment by sgmiriuka** [comment]
ok. A few things: 
-In the README the right path should read /circompara2/src/utils/bash
-Din't realize it is in p2.7. After dealing with it, cannot run the test. I found a circompara file here --> /circompara2/src/utils/bash but not a circompara2. 
-I convert the raw files to python3. Looks that installation went find, not tested. Do you expect any problems, i.e., one of the  9 circ scripts running on 2.7?

**Comment by egaffo** [comment]
I updated the README with additional instructions. I hope now they are more clear.
Yes, I had to stuck with Python2.7 as more than one of the integrated tools (now I do not remember exactly which ones) were not upgraded to Python3 from the original authors and some probably never will (e.g. TopHat2 has unmaintained code). I did try to convert all software to Python3 but it was a mess and I preferred to use a virtual environment. I put some instructions in the latest README. Using Python3 will eventually fail at some point.
